documentation/kubernetes.md

```diff
@@ -29,29 +32,84 @@ spec:
     spec:
       restartPolicy: Always
       containers:
-      - name: app1
+      - name: app
         image: mcr.microsoft.com/dotnet/samples:aspnetapp
```

Comment:
Note this sample app will not support dump calls, per https://github.com/dotnet/dotnet-monitor/issues/2932